src/pages/contract/_components/ContractOptions.tsx

```diff
@@ -0,0 +1,228 @@
+import { Dispatch, SetStateAction } from "react";
+
+interface ContractOptionsProps {
+  isTransactionClicked: boolean;
+  isCodeClicked: boolean;
+  isTokenClicked: boolean;
+  setIsTransactionClicked: Dispatch<SetStateAction<boolean>>;
+  setIsCodeClicked: Dispatch<SetStateAction<boolean>>;
+  setIsTokenClicked: Dispatch<SetStateAction<boolean>>;
+}
+
+export default function ContractOptions({
+  isTransactionClicked,
+  isCodeClicked,
+  isTokenClicked,
+  setIsTransactionClicked,
+  setIsCodeClicked,
+  setIsTokenClicked,
+}: ContractOptionsProps) {
+  return (
+    <div className="flex flex-row gap-x-4 md:pt-[3.67px] pt-[19.67px]">
+      {isCodeClicked ? (
+        <div>
```

Comment:
```suggestion
        <>
```

Reply:
if we use react fragment, the fragment is unable to detect 2nd child div